css/index.css

```diff
@@ -0,0 +1,52 @@
+html {
+    font-size: 100%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+    background-color: #BEE1E6;
+}
+h1 {
+    color: #FFF1E6;
+}
+h2 {
+    color: #FAD2E1;
+}
+section {
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 2%;
+    padding-top: 1%;
+    padding-left: 1%;
+    padding-bottom: 1%;
+    background-color: #DFE7FD;
+}
+#experience_block{
+    width: 80%;
+    margin-left: 10%;
+    padding-top: 1%;
+    background-color: #DFE7FD;
+}
+#message_sec {
+    font-size: 150%;
+    text-align: center;
+    color: blueviolet;
+
+}
+#message_sec input {
+    
+    margin-bottom: 3%;
+    font-size: 50%;
+}
+#message_sec textarea {
+    margin-bottom: 3%;
+    font-size: 50%;
+}
+button:hover {
+    color: #FAD2E1;
+    background-color: #EAE4E9;
+}
+footer {
+    letter-spacing: 5px;
+    background-color: #FDE2E4;
+}
```

Comment:
Changing the spacing means padding or margin, but I understand why you would add letter-spacing. 👍🏼